### ISSUE TITLE
French "Report a problem" button contains English text

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#96]
URL: https://hcdev.openplus.ca/fr/avis-rappel/test-rappel-salades-chou-frise-marque-curation-foods-et-salades-hachees-pretes-servir
User Agent: Chrome
Language: French

Issue Description: Within the "Report a problem" button there is English text on the French page. 
Steps to reproduce:
1. Navigate to the URL
2. Scroll to the bottom of the page and click on the "Signaler un problème ou une erreur sur cette page"
Under the text area, the text should be in French. Currently is in English.
![image](https://user-images.githubusercontent.com/65728700/132500138-9243d373-d3da-4118-89fd-52d89939e33c.png)

Expected results:
Text should be in FR.

**Comment by ndolter** [comment]
Validated in Chrome ver 94.0.4606.61 that the text has been translated when the user is logged in. Please note that the character count does not get displayed if the user is not logged in.  Please advise if this is the way it's designed to be.  If so, the ticket can be closed.
![image](https://user-images.githubusercontent.com/68117281/136081167-05607375-9349-41d2-a23d-3f5929f53c7a.png)

**Comment by echouin** [comment]
Hello, this was completed by OP. I think you may have this in error as you get the count when you are not logged in

**Comment by ndolter** [comment]
Could it perhaps be the version of Firefox?  I currently have the issue in Firefox version 92.0.1 on my VM, but don't have the issue with a different version of FF on my host pc.
![image](https://user-images.githubusercontent.com/68117281/136233594-af1bd6ee-2ad1-4e70-ac30-c46754ae18a0.png)

**Comment by echouin** [comment]
From OP:

﻿I can't duplicate this. The screenshot below is Firefox 92.0 in an windows 10 emulator and it looks ok

![image](https://user-images.githubusercontent.com/88099454/137544860-e6183584-59f9-4b26-bf45-e9970c280146.png)

**Comment by hydegeoff** [comment]
The character counter does not appear for Report a Problem when the user is logged in.  See screenshot...
Browser Used: Chrome v99
![Home-ReportProbrlem- Missing character counter](https://user-images.githubusercontent.com/88147013/159929178-f6673c9a-fd3f-44ec-937a-68be4252c8a5.jpg)

**Comment by davidgotlieb** [comment]
@hydegeoff Is this really an issue though? Why would someone logged into the system use the report a problem feature? This is a public facing feature, does it work when not logged in? Seems very edge use case to me?

**Comment by echouin** [comment]
From OP: the character count does not appear when logged in. You get the count when not logged in.

**Comment by hydegeoff** [comment]
I have changed the priority to minor

**Comment by echouin** [comment]
I no longer see a character counter, as above, this does not appear to happen. this issue can be closed.

![image](https://github.com/user-attachments/assets/cf72b00c-dada-4bfd-97c8-00e12fe8e556)
